content/departments/engineering/dev/tools/scaletesting.md

```diff
@@ -0,0 +1,79 @@
+# ScaleTesting Instance
+
+This page documents resources that are relevant for engineers looking to run Sourcegraph at scale as a preemptive measure to uncover bugs that
+may only be caught when operating under particular scale. Typically, this enables to approximate a customer instance usage pattern to ensure
+that our application will perform within acceptable boundaries in that context.
+
+Join the [#wg-test-at-scale](https://sourcegraph.slack.com/archives/C040LV3PS4C) Slack channel to join conversations about its use, both from theperspective of an engineer using it to test things or to follow or contribute to its development.
+
+## Scaling VS Customer Tiers
+
+It is important to make the distinction that by scale here, we're referring to the scale of operations from the perspective of a customer. So we're talking in terms of number of repositories, their size and the number of users. This is not directly about observing far we can scale horizontally or vertically a given service, though this can be achieved though the same mean.
```

Comment:
```suggestion
It is important to make the distinction that by scale here, we're referring to the scale of operations from the perspective of a customer. So we're talking in terms of number of repositories, their size and the number of users. This is not directly about observing how far we can scale horizontally or vertically a given service, though this can be achieved though the same mean.
```